enferno/admin/views.py

```
@@ -4047,7 +4050,7 @@
                     return HTTPResponse.forbidden("Restricted Access")
             except s3.exceptions.NoSuchKey:
                 return HTTPResponse.not_found("File not found")
-            except Exception as e:
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)